pkg/cmd/admin/policy/cani.go

```diff
@@ -209,13 +226,18 @@ func (o *canIOptions) listAllPermissions() error {
 		if err != nil {
 			return err
 		}
-		rulesReviewStatus = whatCanYouDo.Status
 
+		policyRules = whatCanYouDo.Status.Rules
+		rulesReviewResult = whatCanYouDo
+	}
+
+	if o.Printer != nil {
+		return o.Printer.PrintObj(rulesReviewResult, o.Out)
 	}
 
 	writer := tabwriter.NewWriter(o.Out, tabwriterMinWidth, tabwriterWidth, tabwriterPadding, tabwriterPadChar, tabwriterFlags)
```

Comment:
@juanvallejo just updated the code https://github.com/abstractj/origin/commit/8c5e3775e4f751fc5d5ea3f301aee70940ce3480. But I feel like there's something missing, because it's not working :)

Some considerations:
1. If we remove `kcmdutil.AddPrinterFlags` we lose `--output`. I believe that if that's the idea, might be necessary to add something like `cmd.Flags().BoolVar(&o.Out....`
2. The code duplicates the same code at `DescribePolicyRule`, should we remove this function? I guess not, but worth to check.